src/components/explorer/ExplorerSearchResultsTable.js

```diff
@@ -63,8 +77,10 @@ const ExplorerSearchResultsTable = ({
     }, [dispatch, dataset, activeTab]);
 
     const onPageChange = useCallback((pageObj, filters, sorter) => {
```

Comment:
is this called on filter change too?  in which case maybe we should call this `onTableChange` instead

Reply:
yes, the onPageChange function is called when filters change, in addition to handling pagination changes